apps/user-portal/src/configs/app.ts

```diff
@@ -49,10 +51,12 @@ export const ServiceResourcesEndpoint: ServiceResourcesType = {
     challengeAnswers: `${GlobalConfig.serverHost}/api/users/v1/me/challenge-answers`,
     challenges: `${GlobalConfig.serverHost}/api/users/v1/me/challenges`,
     consents: `${GlobalConfig.serverHost}/api/identity/consent-mgt/v1.0/consents`,
+    federatedAssociation: `${GlobalConfig.serverHost}/api/users/v1/me/federated-associations`,
     fidoEnd: `${GlobalConfig.serverHost}/api/users/v2/me/webauthn/finish-registration`,
     fidoMetaData: `${GlobalConfig.serverHost}/api/users/v2/me/webauthn`,
     fidoStart: `${GlobalConfig.serverHost}/api/users/v2/me/webauthn/start-registration`,
     fidoStartUsernameless: `${GlobalConfig.serverHost}/api/users/v2/me/webauthn/start-usernameless-registration`,
+    identityProviders: `${GlobalConfig.serverHost}/t/carbon.super/api/server/v1/identity-providers`,
```

Comment:
Remove hard coded tenant

Reply:
I have removed it entirely since it's no more needed as we are getting the IdP metadata along with the Federated Associations response itself.